tests/functionaltests/Tests/UIKitTests/UIActivityIndicatorTests.mm

```diff
@@ -63,4 +72,479 @@
+            Microsoft::WRL::ComPtr<IInspectable> inspectable(XamlGetInternalProgressRing(objcwinrt::to_insp(xamlElement)));
+            auto progressRing = objcwinrt::from_insp<Controls::ProgressRing>(inspectable);
+
+            [indicator startAnimating];
+            ASSERT_TRUE(indicator.isAnimating);
+            ASSERT_TRUE(progressRing.IsActive());
+            EXPECT_EQ(1.0, xamlElement.Opacity());
+            EXPECT_EQ(1.0, indicator.alpha);
+            // Visibility should not be altered
+            EXPECT_EQ(winrt::Windows::UI::Xaml::Visibility::Visible, xamlElement.Visibility());
+            ASSERT_FALSE(indicator.hidden);
+
+            [indicator stopAnimating];
+            ASSERT_FALSE(indicator.isAnimating);
+            ASSERT_FALSE(progressRing.IsActive());
+            EXPECT_EQ(0.0, xamlElement.Opacity());
+            EXPECT_EQ(0.0, indicator.alpha);
+            // Visibility should not be altered
+            EXPECT_EQ(winrt::Windows::UI::Xaml::Visibility::Visible, xamlElement.Visibility());
+            ASSERT_FALSE(indicator.hidden);
+
+            [indicator setHidesWhenStopped:false];
+            ASSERT_FALSE(indicator.hidesWhenStopped);
+            EXPECT_EQ(1.0, xamlElement.Opacity());
+            EXPECT_EQ(1.0, indicator.alpha);
+            // Visibility should not be altered
+            EXPECT_EQ(winrt::Windows::UI::Xaml::Visibility::Visible, xamlElement.Visibility());
+            ASSERT_FALSE(indicator.hidden);
+        });
+    }
+
+    TEST_METHOD(UIActivityIndicatorView_SetGrayStyle) {
+        StrongId<UIActivityIndicatorViewController> activityIndicatorVC;
+        activityIndicatorVC.attach([[UIActivityIndicatorViewController alloc] init]);
+
+        UXTestAPI::ViewControllerPresenter testHelper(activityIndicatorVC);
+        UIActivityIndicatorView* indicator = [activityIndicatorVC indicator];
+
+        dispatch_sync(dispatch_get_main_queue(), ^{
+            FrameworkElement xamlElement = [indicator _winrtXamlElement];
+            Microsoft::WRL::ComPtr<IInspectable> inspectable(XamlGetInternalProgressRing(objcwinrt::to_insp(xamlElement)));
+            auto progressRing = objcwinrt::from_insp<Controls::ProgressRing>(inspectable);
+
+            // Change to gray style
+            indicator.activityIndicatorViewStyle = UIActivityIndicatorViewStyleGray;
+            EXPECT_EQ(UIActivityIndicatorViewStyleGray, indicator.activityIndicatorViewStyle);
+
+            auto solidForegroundBrush = progressRing.Foreground().as<Media::SolidColorBrush>();
+            UIColor* expectedForegroundColor = [UIColor colorWithWhite:0.0 alpha:0.45];
+            EXPECT_TRUE(UXTestAPI::IsRGBAEqual(solidForegroundBrush, expectedForegroundColor));
+
+            EXPECT_EQ(expectedNormalSize, progressRing.Height());
+            EXPECT_EQ(expectedNormalSize, progressRing.Width());
+
+            // Frame size should not change
+            EXPECT_EQ(50, xamlElement.Height());
```

Comment:
>50 [](start = 22, length = 2)

Might be good to have a constant for this value as well--consistent with expectedNormalSize